packages/platforms/browser/lib/index.ts

```diff
@@ -1,4 +1,4 @@
-export { DefaultRoutingProvider, default, onSettle } from './browser'
+export { DefaultRoutingProvider, default, onSettle, ReactRouterRoutingProvider } from './browser'
```

Comment:
I think we probably want the latter.  Or it could be `import { ReactRouterRoutingProvider } from '@bugsnag/browser-performance/routing-providers'` (and likewise for e.g the `VueRoutingProvider`). 
Is there any advantage to that?  (Fewer packages to maintain?)